week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,47 +1,82 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
+//main source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators
+//https://www.w3schools.com/js/js_comparisons.asp
+
+//expected test("mood function works", mood() === "I am not happy");
+
 function mood() {
   let isHappy = true;
-
-  if (isHappy) {
+  if (!isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
+/* expected test(
+  "greaterThanTen function works",
+  greaterThan10() === "num is greater than or equal to 10"
+
+-https://www.w3schools.com/nodejs/nodejs_get_started.asp
+
+-https://www.w3schools.com/js/js_arrays.asp
+
+); */
+
+
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
-
-  if (isBigEnough) {
+  let isBigEnough = num;
+  if (isBigEnough >= num) {
     return "num is greater than or equal to 10";
   } else {
     return "num is not big enough";
   }
 }
+// where do i put the console.log(greaterThan10) in this task;
```

Comment:
You could put it here, as long as you make sure to *call* your function, using brackets (`()`), e.g.:

```javascript
console.log(greaterThan10())
```